nni/algorithms/compression/pytorch/pruning/amc/channel_pruning_env.py

```diff
@@ -227,7 +227,7 @@ def step(self, action):
 
     def reset(self):
         # restore env by loading the checkpoint
-        self.pruner.reset(self.checkpoint)
+        self.pruner.reset_bound_model(self.checkpoint)
```

Comment:
Suggest testing this PR on IT.

Reply:
@J-shang please manually trigger IT for this PR